src/content/pages/attend/travel.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Travel Tips & Tricks
+order: 10
+---
+
+
+# Travelling to PyCon AU 2024
+
+For some folks, PyCon AU 2024 will be their first time in Melbourne. Welcome! For others, it might have been a few years so welcome back! Here are some advice for getting around town (and to the conference).
+
+## Interstate & International travel
+Book travel to Melbourne Tullamarine Airport (MEL). Melbourne also has a second airport, Avalon Airport (AVV), which is significantly further from the city and less well-served by public transport.
+
+Both MEL and AVV are served by [SkyBus](https://www.skybus.com.au/), a shuttle bus service that runs regularly between airports and the city. Tickets can be purchased upon arrival but are cheaper ($40 return for an adult) if pre-booked online.
+
+## Public transport
+Melbourne has a single ticketing system for all public transport, including trains, trams, and buses. The system is called [Myki](https://www.ptv.vic.gov.au/tickets/Myki). You can purchase a Myki card at the airport, most train stations, and many convenience stores around town. The card costs $6 and needs to be topped up with money for travel. 
+
+### Trams
+Melbourne has an extensive and easy to use tram network. Travel is free inside [the green free tramzone](https://www.ptv.vic.gov.au/assets/PDFs/Maps/Network-maps/5b4a3efe80/PTV-Free-Tram-Zone-Map.pdf) covering most of the CBD, but you need to 'tap on' with your Myki if travelling outside hte green zone.
+
+### Trains
+Train fares in Melbourne also use the Myki system. The closest train station to the conference venue is Southern Cross Station, which is a major hub for regional and interstate trains. Note that regional and interstate trains are not covered by the Myki system and are ticketed seperately.
+
+# Arriving at PyCon AU 2024
+This year's conference is held in the Goldfields Theatre at the Melbourne Convention and Exhibition Centre (MCEC). The MCEC is located at 1 Convention Centre Place, South Wharf, Melbourne, Victoria, 3006.
+
+MCEC is a large venue with multiple entrances. The Goldfields Theatre is located on the ground floor, accessible from the main entrance on Convention Centre Place. The venue is fully accessible, with ramps and elevators available for those who need them.
+
+We recommend the following routes when arriving via:
+
+## via Tram
+If arriving on routes 35,70 and 75, alight at South Wharf/Wurundjeri Way and its approx 10m walk across Seafarers Bridge.
+
+If arriving on routes 12, 96, 109, alight at Stop 124A Casino/MCEC/Clarendon St (Southbank) and walk via MCEC Exhibition halls to the far end.
+
+Note: The Clarendon St tram stop is outside the free tram zone, and some delegates may enjoy the additional stroll across the bridge for free tram travel to the conference.
```

Comment:
i think this is better split into two sentences next to each prospective route. See comments above.